typegate/src/runtimes/deno/deno_messenger.ts

```diff
@@ -31,7 +31,7 @@ export class DenoMessenger extends LazyAsyncMessenger<Worker, Task, unknown> {
               ...permissions,
               // non-overridable permissions (security between typegraphs)
               run: false,
-              read: false,
+              read: [".", "tmp/scripts/deno"], // cwd for import paths
```

Comment:
why is "." required? can you customise the path per typegraph/runtime otherwise the different runtimes will be able to run others which should not be accepted